enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/søknad/dokumentasjonsbehov/DokumentasjonsbehovDto.kt

```diff
@@ -0,0 +1,12 @@
+package no.nav.familie.kontrakter.ef.søknad.dokumentasjonsbehov
+
+import no.nav.familie.kontrakter.ef.søknad.Dokumentasjonsbehov
+import no.nav.familie.kontrakter.ef.søknad.SøknadType
+import java.time.LocalDateTime
+
+data class DokumentasjonsbehovDto(
+    val dokumentasjonsbehov: List<Dokumentasjonsbehov>,
```

Comment:
Hvor skal denne brukes? Persistering i mottak?

Reply:
Denne er for henting av data fra mottak (soknad-api henter fra mottak)
Pesisteringen skjer via SøknadMedVedlegg